Error with python at alpine 3.12

$ make build_minikube
make[1]: Entering directory '/home/hugo/ws/serlo/infrastructure-images/container/gsutil-base'
docker images | grep serlo/gsutil-base && echo "image serlo/gsutil-base already exists use docker_build" || make docker_build
make[2]: Entering directory '/home/hugo/ws/serlo/infrastructure-images/container/gsutil-base'
docker build --build-arg version=1.1.0  --build-arg git_revision=7abba2d40adef70739da99b114ba5da75752b2ae -t serlo/gsutil-base .
Sending build context to Docker daemon  4.608kB
Step 1/13 : FROM alpine
latest: Pulling from library/alpine
801bfaa63ef2: Pull complete 
Digest: sha256:3c7497bf0c7af93428242d6176e8f7905f2201d8fc5861f45be7a346b5f23436
Status: Downloaded newer image for alpine:latest
 ---> 389fef711851
Step 2/13 : RUN apk update && apk add bash curl python python-dev py-pip build-base && rm -rf /var/cache/apk/*
 ---> Running in df26916602c2
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz
v3.12.3-13-ga43a0bc1d0 [http://dl-cdn.alpinelinux.org/alpine/v3.12/main]
v3.12.3-15-ge843fbbe95 [http://dl-cdn.alpinelinux.org/alpine/v3.12/community]
OK: 12745 distinct packages available
ERROR: unsatisfiable constraints:
  python (missing):
    required by: world[python]
  python-dev (missing):
    required by: world[python-dev]
The command '/bin/sh -c apk update && apk add bash curl python python-dev py-pip build-base && rm -rf /var/cache/apk/*' returned a non-zero code: 2
make[2]: *** [Makefile:15: docker_build] Error 2
make[2]: Leaving directory '/home/hugo/ws/serlo/infrastructure-images/container/gsutil-base'
make[1]: *** [../../mk/dockerci.mk:37: docker_build_minikube] Error 2
make[1]: Leaving directory '/home/hugo/ws/serlo/infrastructure-images/container/gsutil-base'
make: *** [Makefile:20: build_minikube_gsutil-base] Error 2

--- a/container/dbdump/Dockerfile
+++ b/container/dbdump/Dockerfile
@@ -1,5 +1,5 @@
 FROM alpine
-RUN apk update && apk add bash curl python python-dev py-pip build-base && rm -rf /var/cache/apk/*
+RUN apk update && apk add bash curl python3 python3-dev py-pip build-base && rm -rf /var/cache/apk/*
 RUN curl https://sdk.cloud.google.com | bash >/dev/null
 ENV PATH=${PATH}:/root/google-cloud-sdk/bin:/root/google-cloud-sdk/platform/gsutil
 

--- a/container/dbsetup/Dockerfile
+++ b/container/dbsetup/Dockerfile
@@ -1,5 +1,5 @@
 FROM alpine
-RUN apk update && apk add bash curl python python-dev py-pip build-base && rm -rf /var/cache/apk/*
+RUN apk update && apk add bash curl python3 python3-dev py-pip build-base && rm -rf /var/cache/apk/*
 RUN curl https://sdk.cloud.google.com | bash >/dev/null
 ENV PATH=${PATH}:/root/google-cloud-sdk/bin:/root/google-cloud-sdk/platform/gsutil
 

--- a/container/gsutil-base/Dockerfile
+++ b/container/gsutil-base/Dockerfile
@@ -1,5 +1,5 @@
 FROM alpine
-RUN apk update && apk add bash curl python python-dev py-pip build-base && rm -rf /var/cache/apk/*
+RUN apk update && apk add bash curl python3 python3-dev py-pip build-base && rm -rf /var/cache/apk/*
 RUN curl https://sdk.cloud.google.com | bash >/dev/null
 ENV PATH=${PATH}:/root/google-cloud-sdk/bin:/root/google-cloud-sdk/platform/gsutil
 ADD run.sh /tmp/run.sh

